### DISCUSSION TITLE
Expedition 1 limited exo tech slots?

### DISCUSSION THREAD

**Discussion by leftcoastfunk** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#47] [Q&A] [answered]
Apologies if this is answered elsewhere, I couldn't find an answer...

As a player who only very recently started (Exp 14 was my first), I'm now finally going back through old expeditions and currently playing Expedition 1 redux! Are the exosuit tech slots supposed to be limited to max 21 slots only? I know when the expedition first happened things were likely quite different, but I just couldn't tell if this was a bug or a feature. I found the two additional tech components in main inventory, but after playing the expedition a while now and trying to max out tech slots the game is stopping me at 21

thx :)

**Comment by leftcoastfunk** [comment, marked as answer]
uhhhhh ok ok I might be going absolutely crazy. I had tried forever to get past the 21 tech slots the game was limiting me to, but after posting I did a quick save/quit/restart and the same space station suddenly decided I could upgrade further smh

Don't know how to delete so I'll just mark as answered

**Comment by cwmonkey** [comment]
Ah thanks, I actually didn't know about the save/restart thing fixing it. For a while I assumed I had to purchase more non-tech inventory slots before it would unlock those. Then I just started thinking it was random, lol.
